src/main/java/me/jellysquid/mods/sodium/mixin/features/mipmaps/MixinSpriteContents.java

```diff
@@ -0,0 +1,138 @@
+package me.jellysquid.mods.sodium.mixin.features.mipmaps;
+
+import net.minecraft.client.texture.NativeImage;
+import net.minecraft.client.texture.SpriteContents;
+import net.minecraft.util.Identifier;
+import org.objectweb.asm.Opcodes;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Mutable;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.Unique;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+/**
+ * This Mixin is ported from Iris at <a href="https://github.com/IrisShaders/Iris/blob/41095ac23ea0add664afd1b85c414d1f1ed94066/src/main/java/net/coderbot/iris/mixin/bettermipmaps/MixinTextureAtlasSprite.java">MixinTextureAtlasSprite</a>.
+ */
+@Mixin(SpriteContents.class)
+public class MixinSpriteContents {
+    @Mutable
+    @Shadow
+    @Final
+    private NativeImage image;
+    // Generate some color tables for gamma correction.
+    private static final float[] SRGB_TO_LINEAR = new float[256];
```

Comment:
Duplicated code from the other mixin. Maybe we want to move this into a helper class?